src/pages/edit_resource/description_step/TagsField.tsx

```diff
@@ -111,7 +123,15 @@ const TagsField: FC<TagsFieldProps> = ({ setAllChangesSaved }) => {
           )}
         </Field>
         {saveError && <ErrorBanner userNeedsToBeLoggedIn={true} />}
-      </StyledContentWrapper>
+        <HelperTextPopover popoverId="tags-explanation" ariaButtonLabel={t('explanation_text.tags_helper_aria_label')}>
+          <Typography variant="body1">
```

Comment:
Med så mye tekst ble det litt kompakt. Kanskje legge inn et linjeskift eller to ?  -

Reply:
Lagt inn med flere typography og lagt inn litt marin-bottom.